repo/product.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+
+namespace Mercury
+{
+
+    public class product
+    {
+
+        public string p_id = "";
+        public string p_price = "";
+        public string p_quantity = "";
+        public string p_description = "";
+        public string p_image = "";
+        public string created_at = "";
+        public string updated_at = "";
+
+    }
+    }
```

Comment:
is this needed we already have a product model

Reply:
i forget to delete it it is not a part of project